CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 selinux Cookbook CHANGELOG
 ==========================
 
+v0.10.0 (2016-05-21)
```

Comment:
Instead of labeling with a specific version, leave this as UNRELEASED. Ideally, don't modify the CHANGELOG but when you have a large number of commits in one PR it's helpful.